### PR TITLE
Moved bot removal SQL to base_event table creation, and fixed spacing.

### PR DESCRIPTION
**Pull request by sheyd**
#### What's this PR do?
- Fixes bot removal code syntax issue, as well as moving it inline with `public.snowplow_base_events` table creation per [this comment](https://github.com/DoSomething/quasar/pull/785/files#r299689326) since I agree with Megha that it's probably better to put inline from a logic perspective.

#### Where should the reviewer start?
https://github.com/DoSomething/quasar/compare/snowplow-bot-syntax-fix?expand=1#diff-7ddb641c696addd4d88b82589b3fbb65R23

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/165548869

### CODE REVIEW

File: data/sql/derived-tables/snowplow_events.sql [hunk 1]
```diff
@@ -20,18 +20,16 @@ CREATE TABLE public.snowplow_base_event AS
     refr_urlpath AS referrer_path,
     refr_source AS referrer_source
   FROM ft_snowplow."event"
+  WHERE event_id NOT IN
```
Comment (mjain-1) [inline data/sql/derived-tables/snowplow_events.sql]: i think `NOT EXISTS` might be more performant

Comment (sheyd) [reply data/sql/derived-tables/snowplow_events.sql]: 👍 Updated!

File: data/sql/derived-tables/snowplow_events.sql [hunk 2]
```diff
@@ -20,7 +20,7 @@ CREATE TABLE public.snowplow_base_event AS
     refr_urlpath AS referrer_path,
     refr_source AS referrer_source
   FROM ft_snowplow."event"
-  WHERE event_id NOT EXISTS 
+  WHERE NOT EXISTS
```
Comment (mjain-1) [inline data/sql/derived-tables/snowplow_events.sql]: i think the syntax for NOT EXISTS requires an additional condition in the subquery with `WHERE ft.snowplow."event".event_id = u.event_id`

Comment (sheyd) [reply data/sql/derived-tables/snowplow_events.sql]: Hahaha, no worries. Reverting to original form!

File: data/sql/derived-tables/snowplow_events.sql [hunk 3]
```diff
@@ -20,18 +20,16 @@ CREATE TABLE public.snowplow_base_event AS
     refr_urlpath AS referrer_path,
     refr_source AS referrer_source
   FROM ft_snowplow."event"
+  WHERE event_id NOT IN 
+  (SELECT event_id
+   FROM ft_snowplow.ua_parser_context u
+   WHERE u.useragent_family SIMILAR TO 
+   '%(bot|crawl|slurp|spider|archiv|spinn|sniff|seo|audit|survey|pingdom|worm|capture|(browser|screen)shots|analyz|index|thumb|check|facebook|YandexBot|Twitterbot|a_archiver|facebookexternalhit|Bingbot|Googlebot|Baiduspider|360(Spider|User-agent))%');
```
Comment (mjain-1) [inline data/sql/derived-tables/snowplow_events.sql]: semi-colon needs to be removed here

Comment (sheyd) [reply data/sql/derived-tables/snowplow_events.sql]: Yep, removed!

**Pull request merged**